src/analysis/calculus/fderiv/star.lean

```diff
@@ -0,0 +1,97 @@
+/-
+Copyright (c) 2023 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import analysis.calculus.fderiv.linear
+import analysis.calculus.fderiv.comp
+import analysis.normed_space.star.basic
+
+/-!
+# Star operations on derivatives
+
+For detailed documentation of the Fréchet derivative,
+see the module docstring of `analysis/calculus/fderiv/basic.lean`.
+
+This file contains the usual formulas (and existence assertions) for the derivative of the star
+operation. Note that these only apply when the field that the derivative is respect to has a trivial
+star operation; which as should be expected rules out `𝕜 = ℂ`.
+-/
+
+open_locale classical
+
+
+variables {𝕜 : Type*} [nontrivially_normed_field 𝕜] [star_ring 𝕜] [has_trivial_star 𝕜]
+variables {E : Type*} [normed_add_comm_group E] [normed_space 𝕜 E]
+variables {F : Type*} [normed_add_comm_group F] [star_add_monoid F] [normed_space 𝕜 F]
+  [star_module 𝕜 F] [has_continuous_star F]
+
+variables {f : E → F}
+variables {f' : E →L[𝕜] F}
+variables (e : E →L[𝕜] F)
+variables {x : E}
+variables {s : set E}
+variables {L : filter E}
```

Comment:
Using `L` for both the filter and within the `continuous_linear_map` notation is ever so slightly confusing. If this is ubiquitous throughout `fderiv` results, I'm fine with it for consistency, but otherwise can you pick a different variable letter?

Reply:
These variables are copied verbatim from `add.lean`. I agree this is somewhat confusing.